src/core_modules/capture-core-utils/RulesEngine/RulesEngine.js

```diff
@@ -59,10 +59,7 @@ export default class RulesEngine {
         const dateUtils = getDateUtils(momentConverter);
         const variableService = new VariableService(valueProcessor.processValue, dateUtils);
         this.executionService = getExecutionService(onTranslate, variableService, dateUtils);
-        this.onProcessRulesEffects = getRulesEffectsProcessor(
-            this.executionService.convertDataToBaseOutputValue,
-            outputRulesConverterObject,
-        );
+        this.onProcessRulesEffects = getRulesEffectsProcessor(this.executionService.convertDataToBaseOutputValue, outputRulesConverterObject);
```

Comment:
You would probably get an eslint max-len error for this one?

Reply:
yes you right! Now is fixed 👍 Thanks